scripts/release.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env sh
+set -e
+PATH=$PATH:$PWD/bin
+GOPATH=$PWD
+
+SRC=${PWD}/src/github.com/SUSE/cf-plugin-backup
+TAG=$(git -C ${SRC} describe --tags --abbrev=0)
+LAST_TAG=$(git -C ${SRC} describe --tags --abbrev=0 HEAD^)
+NAME=$(basename $(git -C ${SRC} config --get remote.origin.url) .git)
+
+# Show incoming state
+echo NAME: ${NAME}
+echo TAG_: ${TAG}
+echo LAST: ${LAST_TAG}
+
+# Assemble configuration data for `out` to git-release resource.
+echo > release/name ${TAG}
+echo > release/tag  ${TAG}
+echo > release/body "${NAME}: Release ${TAG}"
+echo >> release/body ""
+git -C ${SRC} log --reverse --pretty=%s --merges ${LAST_TAG}...${TAG} \
+    >> release/body
+
+# Build the tarballs for the release
+make -C ${SRC} build dist
+
+# Show configuration in log
+echo Release information:
+for path in release/*
+do
+    printf "%s: " "${path#**/}"
+    cat $path
+done
+
+# Unpack the tarballs into the release.
+mkdir release/assets
+for path in ${SRC}/*.tgz
+do
+    echo Processing ${path} ...
+
+    tar xfz "${path}" \
+	-C release/assets/ \
+	--strip-components=1 \
+	--overwrite \
+	--transform="s@/cf-plugin-backup@/$(basename ${path%%.tgz})@"
```

Comment:
Then this should be `$(basename ${path} .tgz)` again. Sorry about that.

Reply:
ok

Reply:
Done with commit 0105f25412c5920a52ecccf345d1fc1fe7f8b707
The commitish made no change re association.
Looking a bit more now.